File: src/Api/Model/DB/SocioEmpresa.cs
using Newtonsoft.Json;
using Realms;
using System;
using System.Collections.Generic;
using System.Text;

namespace CriadorBaseDados.Model.DB
{
    [JsonObject(MemberSerialization.OptIn)]
    public class SocioEmpresa : RealmObject
    {
        [Ignored]
        [JsonProperty]
        public string CNPJSocio => Socio.CNPJ_CPF;
        public Socio Socio { get; set; }
        [Ignored]
        [JsonProperty]
        public string CnpjEmpresa => Empresa.CNPJ;
        public Empresa Empresa { get; set; }
        [Ignored]
        [JsonProperty]
        public int CodQualificacao => Qualificacao.Cod;
        public QualificacaoResponsavel Qualificacao { get; set; }
        [JsonProperty]
        public int DataEntrada { get; set; }
        [JsonProperty]
        public double Capital { get; set; }
    }
}

File: src/Mobile/b2b/lib/model/core/socio_empresa.dart
class SocioEmpresa {
  final String cnpjSocio;
  final String cnpjEmpresa;
  final int codQualificacao;
  final int dataEntrada;
  final double capital;

  SocioEmpresa(this.cnpjSocio, this.cnpjEmpresa, this.codQualificacao, this.dataEntrada, this.capital);

  SocioEmpresa.fromJson(Map<String, dynamic> json)
      : cnpjSocio = json['cnpjSocio'],
        cnpjEmpresa = json['cnpjEmpresa'],
        codQualificacao = json['codQualificacao'],
        dataEntrada = json['dataEntrada'],
        capital = json['capital'];
}
